### ISSUE TITLE
fraction proposal network of FQF

### ISSUE THREAD

**Issue by zhangpaipai** [ku2482/fqf-iqn-qrdqn.pytorch#19]
Hi, guys!
I have some problems about fraction proposal network of FQF:
1. why set fraction_lr=2.5e-9, which is very small? And I found that the tau_hats distibution almost had no change during the training.
2. why apply initialize_weights_xavier(x, gain=0.01)? When I trained, I found if I didn't apply this initialization, gradient explosion would happen sometiomes.
3. why use RMSprop, and set alpha=0.95, eps=0.00001, of which the default values are 0.99 and 10e-8 respectively.
4. And I found that the tau_hats distibution almost had no change during the training of qbert. Is it the key of this algorithm?
thanks!

**Comment by toshikwa** [comment]
Hi @zhangpaipai 

I have just implemented and reproduced the algorithm described in the paper, and I didn't design the algorithm.
I am sorry, but please ask the authors of the paper.
